neuralpp/inference/graphical_model/representation/factor/symbolic_factor.py

```diff
@@ -130,11 +134,15 @@ def _dict_to_context(self, assignment_dict: dict) -> SymPyExpression:
         return result
 
     def _any_function_of_arity(self, expressions: List[Expression]) -> Callable:
+        """
+        This is currently used because there is a bug in FunctionApplication callable for mul and div.
+        Will be removed once the bug is fixed.
+        """
         arity = 0
         for expression in expressions:
             if isinstance(expression, FunctionApplication):
                 arity += expression.number_of_arguments
             else:
                 arity += 1
 
-        return Callable[[Any for i in range(arity)], Any]
+        return Callable[[[Any] * arity], Any]
```

Comment:
There is an extra bracket here. This is going to turn into `Callable[[[Any, ..., Any]], Any]`. It needs to be `Callable[[Any] * arity, Any]`.